### PR TITLE
minor updates

### PR DESCRIPTION
**Pull request by fordie**

### CODE REVIEW

File: conf/app.routes [hunk 1]
```diff
@@ -10,7 +10,7 @@ GET        /apply-for-help-to-save/how-the-account-works    @uk.gov.hmrc.helptos
 GET        /apply-for-help-to-save/how-we-calculate-bonuses @uk.gov.hmrc.helptosavefrontend.controllers.IntroductionController.getHowWeCalculateBonuses
 GET        /apply-for-help-to-save/apply                    @uk.gov.hmrc.helptosavefrontend.controllers.IntroductionController.getApply
 
-GET        /check-and-confirm-your-email                    @uk.gov.hmrc.helptosavefrontend.controllers.RegisterController.getConfirmEmailPage
+GET        /select-email                                    @uk.gov.hmrc.helptosavefrontend.controllers.RegisterController.getConfirmEmailPage
```
Comment (sureshhmrc) [inline conf/app.routes]: if you change this, you should change all its occurances in the message file as well, for ex:
`hts.check-and-confirm-your-email.title`

File: app/uk/gov/hmrc/helptosavefrontend/views/register/check_your_email.scala.html [hunk 2]
```diff
@@ -20,7 +20,7 @@
 
 @(email: String, oldEmail: Option[String])(implicit htsContext: HtsContext, request: Request[_], messages: Messages)
 @uk.gov.hmrc.helptosavefrontend.views.html.main_template(
-    title = messages("hts.introduction.title"),
+    title = "Verify your email address - Help to Save - GOV.UK",
```
Comment (AndyHWChung) [inline app/uk/gov/hmrc/helptosavefrontend/views/register/check_your_email.scala.html]: Do we want to put page titles in the messages file?

**Comment by fordie** [comment]
@AndyHWChung @sureshhmrc can you take another look please?

File: app/uk/gov/hmrc/helptosavefrontend/views/register/confirm_email.scala.html [hunk 3]
```diff
@@ -23,19 +23,19 @@
 
 @(email: String, form: Form[ConfirmEmail])(implicit htsContext: HtsContext, request: Request[_], messages: Messages)
 
-@uk.gov.hmrc.helptosavefrontend.views.html.main_template(title = "confirm-email", bodyClasses = None) {
+@uk.gov.hmrc.helptosavefrontend.views.html.main_template(title = "Do you want to use the email address we hold for you? - Help to Save - GOV.UK", bodyClasses = None) {
```
Comment (sureshhmrc) [inline app/uk/gov/hmrc/helptosavefrontend/views/register/confirm_email.scala.html]: this message should also go into messages file ?

Comment (AndyHWChung) [reply app/uk/gov/hmrc/helptosavefrontend/views/register/confirm_email.scala.html]: ditto

**Comment by AndyHWChung** [comment]
LGTM - @sureshhmrc?

**Comment by sureshhmrc** [comment]
LGTM

**Pull request merged**